exporter/redis.go

```diff
@@ -408,6 +408,8 @@ func (e *Exporter) scrape(scrapes chan<- scrapeResult) {
 		var c redis.Conn
 		var err error
 
+		scrapes <- scrapeResult{Name: "up", Addr: addr, Value: 0}
```

Comment:
I suppose the exporter must not publish partial metrics i.e. if /metrics is called in the middle of scrape(). As far as I understand, metrics updated after scrape() exists so redis_up will be updated correctly.

Correct me if I'm wrong